streamly-process.cabal

```diff
@@ -1,96 +1,103 @@
 cabal-version:       2.2
 name:                streamly-process
 version:             0.1.0.0
-synopsis:            Stream Interfaces For System Processes
+synopsis:            Streaming interfaces for system processes
 description:
-  Streamly is a framework for writing programs in a high level, declarative
-  data flow programming paradigm. It provides a simple API, very close to
-  standard Haskell lists.
-  .
-  This package allows us to extract streams from Processes
-  .
-homepage:            https://github.com/composewell/streamly-process
+  Connect system processes with Haskell functions or with each other
+  like Unix pipes using streamly streams.
+homepage:            https://streamly.composewell.com
 bug-reports:         https://github.com/composewell/streamly-process/issues
 license:             Apache-2.0
 license-file:        LICENSE
 author:              Composewell Technologies
 maintainer:          streamly@composewell.com
 copyright:           Composewell Technologies
 category:            Control, Concurrency, Streaming, Processes
+stability:           Experimental
 build-type:          Simple
 extra-source-files:
-    .github/workflows/haskell.yml
-    .gitignore
-    .hlint.yaml
-    appveyor.yml
-    stack.yaml
-    CHANGELOG.md
-    README.md
-    NOTICE
-    cabal.project.ci
-    design/proposal.md
+  CHANGELOG.md
+  NOTICE
+  README.md
+  design/proposal.md
```

Comment:
This is cleaner. We should make it a habit to add only the necessary files to the dist.